horace_core/sqw/file_io/@combine_sqw_pix_job/private/read_pix_for_nbins_block_.m

```diff
@@ -36,6 +36,16 @@
 
 %
 bin_filled = false(n_bin2_process,1);
+if ischar(run_label)
```

Comment:
Are there tests that run with/without `nochange` runlabel to confirm the change from passing the n-th letter of the word to the n-th filenum works?

Reply:
I do not think so. I am actually not sure that there are separate tests for `write_nsqw_to_sqw` at all. Have to check

Reply:
No separate tests -- only as part of gen_sqw